### PR TITLE
include @finos/perspective-webpack-plugin by default

### PR DESCRIPTION
**Pull request by threepointone**
This PR adds `@finos/perspective-webpack-plugin` to the webpack plugin for apps. This makes modular usable by a wider range of teams inside JPM, considering how many of them use perspective. This may be a temporary addition until we figure out how to have better builds from perspetive themselves directly. Further, this doesn't add it for rollup/library builds, so we're not going to advertiswe this too loudly just yet,

I added the sample perspective react app's code (https://github.com/finos/perspective/tree/master/examples/react) into our own sample app to verify that this works. It does!

### CODE REVIEW

**Comment by texodus** [comment]
@threepointone If you're interested, starting in `0.6.1` we're aiming to make perspective practically configurable without a custom plugin (though `@finos/perspective-webpack-plugin` will continue to exist and will just add `worker-loader` and `file-loader`).  See https://github.com/finos/perspective/pull/1292

Perspective should default work without the plugin for all recent releases as well via its inline-build, which is the package's [`"module"` and `"browser"` fields](https://github.com/finos/perspective/blob/master/packages/perspective/package.json#L8);  in this build the WebAssembly asset is inlined as a base64-encoded string within the Webpack bundle.  This bloats the bundle a bit but requires no Webpack magic and has no other performance impact. See https://github.com/finos/perspective/pull/870

If you need any specific changes, questions, etc. re building Perspective - feel free to open a Perspective Issue or @ me, we are constantly seeking to improve it.

**Comment by threepointone** [comment]
Thanks @texodus! I'll reach out to you soon.

File: packages/modular-site/src/App.tsx [hunk 1]
```diff
@@ -1,3 +1,56 @@
...
+          void viewer.current.restore(config);
+        }
+      },
+      (err) => console.error(err),
+    );
+  }, []);
+
+  // You can also the use the stringified config values as attributes
+  return (
+    //eslint-disable-next-line react/react-in-jsx-scope
```
Comment (threepointone) [inline packages/modular-site/src/App.tsx:51]: we need to disable this in our eslint config too, this isn't needed anymore with the new jsx runtime

Comment (LukeSheard) [reply packages/modular-site/src/App.tsx:50]: Woah funky this is gonna change a lot!

**Comment by LukeSheard** [comment]
This is pretty pragmatic - obviously if we could do this without a plugin that would be the best in the long term as this currently locks us into `webpack` for now though.

File: packages/modular-site/src/App.tsx (hunk 1 above)
Comment (threepointone) [reply packages/modular-site/src/App.tsx:50]: Full on! Thankfully there's a codemod, that should be a fun exercise. createElement still works too, so it's backward-compatible.

**Review by LukeSheard — APPROVED** [review]

**Pull request merged**